config/ranks/military.txt

```diff
@@ -1,9 +1,9 @@
 Captain=CPT
-Executive Officer=LT
-Head of Security=WO
-Chief Engineer=WO
-Research Director=WO
-Chief Medical Officer=WO
+Executive Officer=CDR
+Head of Security=LT
```

Comment:
WO specialty: Security Technician
Why would the HOS be lower-ranking than the rest of the heads?